packages/gatsby/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@vercel/gatsby",
+  "version": "0.1.0",
```

Comment:
```suggestion
  "version": "1.0.0",
```

Reply:
We can do this during the publish step, no?